doc/manual/src/language/advanced-attributes.md

```diff
@@ -188,9 +188,12 @@ Derivations can declare some infrequently used optional attributes.
     }
     ```
 
-    The `outputHashAlgo` attribute specifies the hash algorithm used to
-    compute the hash. It can currently be `"sha1"`, `"sha256"` or
-    `"sha512"`.
+    The `outputHash` attribute must be a string containing the hash in either hexadecimal or base-32 notation, or following the format for integrity metadata as defined by [SRI](https://www.w3.org/TR/SRI/).
```

Comment:
The base-32 notation is called `nix32` now to disambiguate it from other base-32 character sets. Maybe you can link from here to [the `convertHash` documentation](https://nixos.org/manual/nix/stable/language/builtins#builtins-convertHash) since it has some examples.

Reply:
Thanks for the info! I added some more information and added a link to `convertHash`.